lib/manager/gradle/shallow/tokenizer.ts

```diff
@@ -35,7 +35,7 @@ const lexer = moo.states({
     [TokenType.Comma]: ',',
     [TokenType.Operator]: /(?:==|\+=?|-=?|\/=?|\*\*?|\.+|:)/, // TODO #12870
     [TokenType.Assignment]: '=',
-    [TokenType.Word]: { match: /[a-zA-Z$_][a-zA-Z0-9$_]+/ }, // TODO #12870
+    [TokenType.Word]: { match: /[a-zA-Z$_][a-zA-Z0-9$_]*/ }, // TODO #12870
```

Comment:
Are you sure about allow empty string?

Reply:
It's not empty, it's `one alpha PLUS zero or more alphanumeric`